posthog/batch_exports/http.py

```diff
@@ -167,51 +156,44 @@ class Meta:
             "last_paused_at",
             "start_at",
             "end_at",
-            "trigger_immediately",
-        ]
-        read_only_fields = [
-            "id",
-            "paused",
-            "created_at",
-            "last_updated_at",
+            "latest_runs",
         ]
+        read_only_fields = ["id", "created_at", "last_updated_at", "latest_runs"]
 
     def create(self, validated_data: dict) -> BatchExport:
         """Create a BatchExport."""
         destination_data = validated_data.pop("destination")
         team_id = self.context["team_id"]
-        interval = validated_data.pop("interval")
-        name = validated_data.pop("name")
-        start_at = validated_data.get("start_at", None)
-        end_at = validated_data.get("end_at", None)
-        trigger_immediately = validated_data.get("trigger_immediately", False)
```

Comment:
`trigger_immediately` was used for a test that is waiting for runs to appear. Since we removed it, test is now failing.